lib/rmt/repo_manager.rb

```diff
@@ -0,0 +1,115 @@
+require 'rmt'
+require 'optparse'
+
+class RMT::RepoManager
+
+  def initialize(argv, out = nil)
+    @argv = argv
+    @options = {}
+    @affected_repos = 0
+    @out = out || StringIO.new
+  end
+
+  def parse_options
+    opt_parser = OptionParser.new do |opts|
```

Comment:
In the long run, I hope (and will push for) that we will use [Thor](http://whatisthor.com/) (or similar) for this. Because the current interface is not good enough IMHO. Example: `--enable` and `--disable` should not be options (no dashes) but commands. And it should not be required to have -r or -p as an hidden requirement for both. So you would use it like: `rmt-repos enable 123` or `rmt-repos disable foo/bar/baz`
With OptionParser that could get quite messy. With Thor, it should be fairly easy.

I'll prepare a proposal (or at least collect my complaints) in the [Trello card](https://trello.com/c/MQAogurq) about CLI.